base/applications/regedit/childwnd.c

```diff
@@ -1,20 +1,21 @@
 /*
- * Regedit child window
- *
- * Copyright (C) 2002 Robert Dickenson <robd@reactos.org>
- * Copyright (C) 2024 Katayama Hirofumi MZ <katayama.hirofumi.mz@gmail.com>
- * LICENSE: LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
+ * PROJECT:     ReactOS Registry Editor
+ * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
+ * PURPOSE:     Frame window (not child window, the child window is merged to the frame window)
```

Comment:
Then why is this still a distinction between the framewnd and childwnd.c files? (OK maybe it could make sense to merge both files together , but once you've got all the flichering fixes done and the PR has been reviewed a 1st time :thinking: )

Reply:
`framewnd.c` and `childwnd.c` are merged.